modules/core/src/types/layer-props.ts

```diff
@@ -122,9 +122,9 @@ export type LayerProps<DataType = any> = {
    */
   updateTriggers?: Record<string, any>;
   /**
-   * The purpose of the layer
+   * The purpose of the layer. `+` separated list of OPERATION enums
    */
-  operation?: 'draw' | 'mask';
+  operation?: string;
```

Comment:
Shame to lose the type here, could we do something like https://stackoverflow.com/questions/75174522/concatenate-string-combinations-in-typescript

Reply:
Can't we offer an array instead? 
```suggestion
  operation?: Operation | Operation[];
  operation?: ('draw' | 'mask' | 'terrain')[]
```
Or if only some combos are valid, offer them in the original type? `draw-terrain` etc

Reply:
GeoJsonLayer's `pointType` uses the `+` separated list convention.